### ISSUE TITLE
sp_BlitzCache - Warn about compilation metrics

### ISSUE THREAD

**Issue by BlitzErik** [BrentOzarULTD/SQL-Server-First-Responder-Kit#1706]
We collect compile time, CPU, and memory from plan XML. I would like to warn on when these metrics are "high".

What's high? Great question!
 - For compile time or CPU, we flag anything over 5 seconds in sp_BlitzQueryStore
 - For compile memory, maybe anything > 1% of max server memory?

Feel free to discuss!

**Comment by rabryst** [comment]
1% of max server memory is probably fine assuming that setting has been configured. I'd also do a fixed lower bound for servers with under 128 GB physical RAM, of say 1 GB. Then for 2016+, replace with info from that DMV.

**Comment by BlitzErik** [comment]
@rabryst Yeah, was thinking total server memory or max server memory, whichever is lower. That work?

**Comment by rabryst** [comment]
Yup.

**Comment by BlitzErik** [comment]
Taking a slightly different approach to this. The plan XML has a `MaxCompileMemory ` attribute that I think I'm going to use instead.
